release-notes/aws/v9.2.5.md

```diff
@@ -0,0 +1,18 @@
+## :zap: Giant Swarm Release 9.2.5 for AWS :zap:
+
+**If you are upgrading from 9.2.4, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release improves the reliability of chart-operator.
```

Comment:
@rossf7 if you have better summary, open to suggestions

Reply:
Hi Stevo, I think this is fine. The changes are bug fixes and stability improvements.

cc @cokiengchiara 

The Helm 3 release when we finally get to it will be more exciting.